Assets/Scripts/UI/Views/Popups/Pause/PauseMenu.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace UI
+{
+    public class PauseMenu : PopupBase
+    {
+        [SerializeField] private Button _closeButton;
+        
+        private IUIRoot _uiRoot;
+        private PauseManager _pauseManager;
+        
+        private void OnEnable()
+        {
+            _closeButton.onClick.AddListener(Pause);
+        }
+
+        private void OnDisable()
+        {
+            _closeButton.onClick.RemoveListener(Pause);
+        }
+
+        private void Pause()
+        {
+            _pauseManager.Paused = !_pauseManager.Paused;
```

Comment:
Pls set "false" manualy. Because its correct behaviour for close pause menu.
We can get error behaviour if _pauseManager.Paused be in False state

Reply:
Done